perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/common.hpp

```diff
@@ -0,0 +1,148 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef AUTOWARE__TENSORRT_BEVDET__COMMON_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__COMMON_HPP_
+
+#include "rclcpp/logger.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+#include <NvInfer.h>
+#include <assert.h>
+#include <cuda_runtime.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <thrust/sort.h>
+
+#include <iostream>
+#include <string>
+
+typedef unsigned char uchar;
+
+#define NUM_THREADS 512
```

Comment:
Avoid using preprocessor macros like `NUM_THREADS`. Prefer using `constexpr` variables for constants to improve readability and maintainability.  Macros can cause issues with debugging and code analysis.